### ISSUE TITLE
Apple M1 Support in GsDevKit

### ISSUE THREAD

**Issue by JupiterJones** [GsDevKit/GsDevKit_home#315]

**Comment by dalehenrich** [comment]
What specifically is needed to support the Apple M1?

**Comment by dalehenrich** [comment]
ah, I think you must be referring to the fact that we need to update the GemStone downloads to recognize the M1 and download the correct version ...

**Comment by JupiterJones** [comment]
Yes, downloads and installOsPrereqs are the ones I've found so far. Not sure if there are others that are platform dependent.

**Comment by JupiterJones** [comment]
I've tested it in my narrow environment (i386 OS10.14.6 and arm64 OS11.2.3) and it's working.

However, there's been a change in the version of xpath between my two test environments that requires different args. I put in a switch that changes at OS11 however, it may have been updated in a minor version in between. Regardless, if someone has an issue, the switch can easily be updated.

**Comment by dalehenrich** [comment]
Sounds perfect! Haha, xpath is a pain in the butt ... I think that those are the only two that need changes ... BTW, I'm taking a crack at using github actions to run tests for the superDoit project ... once I have an idea of how it's working out, I think that GsDevKit_2 and GsDevKit_home should be converted to use them as well ... travis has been "going away in a few weeks" for several months now, but eventually it will, so it's time to learn ...
